File: 프로그래머스/0/181874. A 강조하기/A 강조하기.js
function solution(myString) {
    var answer = '';
    answer = [...myString].map((v) => {
        if(v === 'a') v = 'A';
        else if(v !== 'A' && v == v.toUpperCase()) v = v.toLowerCase();
        return v;
    }).join('');
    return answer;
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.py
def solution(myString):
    myString = list(myString)
    for i in range(len(myString)):
        if myString[i] == "a": myString[i] = "A"
        elif myString[i] != "A": myString[i] = myString[i].lower()
    return "".join(myString)